lib/src/Gren.js

```diff
@@ -913,6 +914,27 @@ class Gren {
         );
     }
 
+    /**
+     * Check that the require tags are exists in tags
+     *
+     * @param {Array} tags
+     * @param {any} requireTags
+     *
+     * @return{}
+     */
+    _validateRequiredTagsExists(tags, requireTags) {
+        if (requireTags.indexOf('all') >= 0 || !(requireTags instanceof Array)) return;
+
+        const tagsNames = tags.map(x => x.tag.name);
+
+        const missingTags = requireTags.filter(x => tagsNames.indexOf(x) < 0);
+        if (missingTags.length > 0) {
+            const notFoundMessage = (missingTags.length === 1) ? 'tag is' : 'tags are';
+            throw chalk.red(`\nThe following ${notFoundMessage} not found in the repository: ${missingTags}. ` +
+                'please provider tags that are exists in the repository');
```

Comment:
Why not everything in the template literal?

Reply:
I'm sorry, I don't understand, could you please explain what you mean?